apps/wizarr-backend/wizarr_backend/helpers/emby.py

```diff
@@ -287,6 +287,16 @@ def invite_emby_user(username: str, password: str, code: str, server_api_key: Op
     else:
         new_policy["EnableLiveTvAccess"] = False
 
+    # Set the hidden user status
+    if invitation.hide_user is not None and invitation.hide_user == True:
+        new_policy["IsHidden"] = True
+        new_policy["IsHiddenRemotely"] = True
+        new_policy["IsHiddenFromUnusedDevices"] = True
+    else:
```

Comment:
Include these defaults in the object creation.

Reply:
updated to show this and removed extra permissions